main.go

```diff
@@ -294,6 +300,10 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	c.batchCollectMetrics(ch, resources)
 }
 
+func getTagID(tag config.ResourceTag) string {
+	return tag.ResourceTagName + "=" + tag.ResourceTagValue
```

Comment:
As your not taking `resource_type` into account, this can not work.

Reply:
Yes ... right ... the filter is done inside the method, that's why I thought I should pass the reference to the cache ... my bad